geode-management/src/main/java/org/apache/geode/management/internal/api/ClusterManagementResult.java

```diff
@@ -17,30 +17,71 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-
-
 public class ClusterManagementResult {
```

Comment:
I think we need to reconsider the organization of the business logic in this class.

Right now the `ClusterManagementResult` is the return object for the `ClusterManagementService` interface. This interface has a `create` `delete` and `update` which will presumably used for many different management things such as managing gateway receivers or senders. I think we're packing in a lot of custom logic for the purposes of create region which may not appropriately apply to the generic use cases that this interface needs to serve.